qemu/tests/rebase_onto_no_backing_file.py

```diff
@@ -0,0 +1,97 @@
+import logging
+import json
+
+from virttest import data_dir
+from virttest.qemu_storage import QemuImg
+
+from qemu.tests.qemu_disk_img import QemuImgTest
+from qemu.tests.qemu_disk_img import generate_base_snapshot_pair
+
+
+def run(test, params, env):
+    """
+    Rebase a qcow2 snapshot onto no backing file.
+
+    1. create a qcow2 snapshot based on a raw image with compat mode 0.10/1.1
+    2. boot the guest from the base
+    3. create a file in the base disk, calculate its md5sum
+    4. shut the guest down
+    5. rebase the snapshot to null to remove the raw backing file
```

Comment:
the statement is inaccuracy. "" is an actual string, although it is an empty one. But null means a variable points to nothing. When make a qcow2 image from snapshot to a ordinary image file, the argument for rebase should be "-b" + empty string.

Reply:
rebase [-f fmt] [-t cache] [-T src_cache] [-p] [-u] -b backing_file [-F backing_fmt] filename
    Changes the backing file of an image. Only the formats "qcow2" and "qed" support changing the backing file.
    The backing file is changed to backing_file and (if the image format of filename supports this) the backing file format is changed to backing_fmt. If backing_file is specified as "" (the empty string), then the image is rebased onto no backing file (i.e. it will exist independently of any backing file).